test/avocado/selenium-login.py

```diff
@@ -134,23 +174,24 @@ def login(self, tmpuser=user, tmppasswd=passwd):
         elem = self.wait_id('login-password-input')
         elem.clear()
         elem.send_keys(tmppasswd)
-        self.wait_id("login-button").click()
+        self.click(self.wait_id("login-button", cond=clickable))
         return elem
 
     def logout(self):
-        elem = self.wait_id('navbar-dropdown')
-        elem.click()
-        elem = self.wait_id('go-logout')
-        elem.click()
+        elem = self.wait_id('navbar-dropdown', cond=clickable)
```

Comment:
Yes, I can clean these cases. Sometimes I've used it because I need shorter prgram line, or I used it in previous version, but using of this element dropped later